components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -246,9 +246,13 @@ type CurrentOrgEntryProps = {
 };
 const CurrentOrgEntry: FunctionComponent<CurrentOrgEntryProps> = ({ title, subtitle }) => {
     return (
-        <div className="w-full text-gray-400 flex flex-col">
-            <span className="text-gray-800 dark:text-gray-300 text-base font-semibold">{title}</span>
-            <span>{subtitle}</span>
+        <div className="w-full text-gray-400 flex items-center justify-between">
+            <div className="flex flex-col">
+                <span className="text-gray-800 dark:text-gray-300 text-base font-semibold">{title}</span>
+                <span>{subtitle}</span>
+            </div>
+            {/* TODO: Replace this with an SVG icon */}
```

Comment:
**praise**: Thanks for adding this placeholder here! Opened https://github.com/gitpod-io/gitpod/issues/16434 to keep track of this and marked it also as a good-first-issue in case anyone from the community would like to work on it.